csharp/.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
```

Comment:
There is already an `.editorconfig` file in the repository root that you can use.